src/internal/tester/resource_owners.go

```diff
@@ -197,3 +197,15 @@ func GetM365SiteID(ctx context.Context) string {
 
 	return strings.ToLower(cfg[TestCfgSiteID])
 }
+
+// UnlicensedM365UserID returns an userID string representing the m365UserID
+// described by either the env var CORSO_M365_TEST_UNLICENSED_USER, the
+// corso_test.toml config file or the default value (in that order of priority).
+// The default is a last-attempt fallback that will only work on alcion's
+// testing org.
+func UnlicensedM365UserID(t *testing.T) string {
+	cfg, err := readTestConfig()
+	require.NoError(t, err, "retrieving unlicensed m365 user id from test configuration", clues.ToCore(err))
```

Comment:
You have to add a formatting marker if you add both a message and a errCore.
```suggestion
	require.NoError(t, err, "retrieving unlicensed m365 user id from test configuration: %+v", clues.ToCore(err))
```

Reply:
If that is the case we might have to update it across the repo. Fixing it here and will raise another PR for all the other places.